UnityProject/Assets/Scripts/Items/FloorHazard.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Mirror;
+using Items;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using Random = UnityEngine.Random;
+
+public class FloorHazard : NetworkBehaviour {
+    public DamageType damageType;
+    public AttackType attackType;
+    public float damage;
+    public bool ignoresShoes = false;
+    public bool ignoresWalking = false;
+    public bool requiresArming = false;
+    public bool armed = false;
+    public float chancePercent = 100f;
+    private BodyPartType[] legs = {BodyPartType.RightLeg,BodyPartType.LeftLeg};
+
+    public void ArmHazard(){
+        armed = true;
+    }
+
+    public void DisarmHazard(){
+        armed = false;
+    }
+
+    public void OnEnterableEnter(BaseEventData eventData)
+	{
```

Comment:
Could you start the { on the next line, just looks better imo.

Also the OnEnterableEnter seems to have weird indenting

Reply:
yeahh, i wont lie i dont really think about indentation alot

Reply:
Can we get a linter and have some github hooks auto format on commit? These nit pick comments on PRs don't provide value to the review.

Reply:
set something up and pr it

Reply:
> Can we get a linter and have some github hooks auto format on commit? These nit pick comments on PRs don't provide value to the review.

they're not valuable code-wise but i still find them valuable